addon/components/pie-chart.js

```diff
@@ -83,7 +88,7 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         color: d.color,
         label: d.label,
         value: d.value,
-        percent: d3.round(100.0 * d.value / total, this.get('maxDecimalPlace'))
+        percent: 100.0*d.value / total
```

Comment:
Ahh sigfig configurations are a great idea actually! I definitely see the value, more for scientific than financial situations though. I think it should probably be in a separate PR though, since this one is mostly just solving a rounding bug + 0% slices issue.